components/header/Header.tsx

```diff
@@ -1,25 +1,48 @@
 import React from 'react';
-import Search from './components/Search';
-import HeaderNav from './components/HeaderNav';
+import Link from 'next/link';
+
+// Compoments
 import DotcmsLogo from './components/DotcmsLogo';
+import HamburgerButton from './components/HamburgerIcon';
+import HeaderButton from './components/HeaderButton';
+import HeaderNav from './components/HeaderNav';
 import HeaderSelect from './components/HeaderSelect';
+import Search from './components/Search';
+import TocIcon from './components/TocIcon';
 
 export const Header = (): JSX.Element => {
+    const buttonIconMobileBarClasses = [
+        'border-b',
+        'border-secondary',
+        'items-center',
+        'justify-center'
+    ];
+    const hamburherIcon = ['justify-end'];
     return (
-        <header className="pt-4 px-6 bg-header border-b">
-            <div className="flex justify-left md:justify-center items-center mb-5 border-gray-100">
-                <div className="w-2/12">
-                    <DotcmsLogo />
-                </div>
-                <div className="hidden justify-center md:flex lg:justify-start w-7/12">
-                    <Search />
-                </div>
-                <div className="hidden md:flex justify-end items-center w-3/12">
-                    <HeaderSelect />
-                </div>
+        <header className="bg-purple flex flex-col justify-between h-32 pt-6 lg:flex-row lg:items-start">
```

Comment:
In mobile the logo and the burger button are not centered and that's because you are using padding to try to center, you need to center by using flexbox align, that way you don't have to care to readjust the padding if the design change and the header height grows.

And and then in larger screen since you don't need to vertically center you use padding.

Reply:
Also the buttons are not working.